### PR TITLE
Add check for nsys gpu module timings report

### PR DESCRIPTION
**Pull request by gartung**

### CODE REVIEW

**Comment by gartung** [comment]
please test

**Comment by smuzaffar** [comment]
looks good
PR tests are not going to run this code. I am merging it and one the results are available then I will merge  https://github.com/cms-sw/cmssdt-ib/pull/26

File: report-summary-merged-prs [hunk 1]
```diff
@@ -1468,6 +1468,7 @@ if __name__ == "__main__":
     MAGIC_COMMAND_FIND_PROFILING = 'test -d ' + JENKINS_ARTIFACTS_DIR + '/profiling/RELEASE_NAME'
     MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER1 = 'ls '+JENKINS_ARTIFACTS_DIR+'/profiling/RELEASE_NAME/ARCHITECTURE/WORKFLOW/step3_*.resources.json 2>/dev/null | head -1 | sed "s|.*/RELEASE_NAME/||;s|.json$||"'
     MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER2 = 'test -s ' + JENKINS_ARTIFACTS_DIR + '/igprof/RELEASE_NAME/ARCHITECTURE/profiling/WORKFLOW/sorted_RES_CPU_step3.txt && echo ARCHITECTURE/profiling/WORKFLOW/sorted_RES_CPU_step3.txt'
+    MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER3 = 'test -s ' + JENKINS_ARTIFACTS_DIR + '/igprof/RELEASE_NAME/ARCHITECTURE/profiling/WORKFLOW/step3_gpu_nsys.txt && echo ARCHITECTURE/profiling/WORKFLOW/step3_gpu_nsys.txt'
```
[the remark this answers is not in the document]
Comment (gartung) [reply report-summary-merged-prs:1471]: yes

[the remark this answers is not in the document]
Comment (gartung) [reply report-summary-merged-prs:1471]: I mean no.

[the remark this answers is not in the document]
Comment (gartung) [reply report-summary-merged-prs:1471]: I had the path correct in the second part of this line.

[the remark this answers is not in the document]
Comment (gartung) [reply report-summary-merged-prs:1471]: Submitted a fix in #1998

**Pull request merged**